tests/test.py

```diff
@@ -32,6 +32,13 @@ def test_run_lf(self):
         results = pp.loadflow.run_dc(n, parameters)
         self.assertEqual(1, len(results))
 
+    def test_new_load(self):
+        n = pp.network.create_eurostag_tutorial_example1_network()
+        n.new_load('VLLOAD', 'LOAD2', double_map={"p0" : 1.1, "q0" : 2.1}, str_map={'connectable_bus':'NLOAD', 'bus': 'NLOAD'})
```

Comment:
Why not doing it using a dataframe update? that would be more consistent with what has already been done.